<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Task</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_new_task">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>130</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>New Task</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_remove_task">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>230</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Remove Task</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_run_task">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>330</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Run Task</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbl_task">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>481</width>
      <height>511</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <column/>
    <column/>
    <column/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_new_task</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>create_task()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>563</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>751</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_remove_task</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>remove_task()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>591</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>768</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_run_task</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>run_task()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>547</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>807</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbl_task</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>select_item()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>786</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>create_task()</slot>
  <slot>remove_task()</slot>
  <slot>run_task()</slot>
  <slot>select_item()</slot>
 </slots>
</ui>
